skimage/io/collection.py

```diff
@@ -157,7 +157,24 @@ def vidread_step(f, step):
       x = ic[0]  # calls vidread_step(video_file, step=3)
       x[5]  # is the sixth element of a list of length 8 (24 / 3)
 
-    Another use of ``load_func`` would be to convert all images to ``uint8``::
+    Alternatively, if a `load_func` is provided and `load_pattern` is a
+    sequence, an ImageCollection of corresponding length will be created,
+    and the individual images are loaded by calling `load_func` with the
```

Comment:
```suggestion
    and the individual images will be loaded by calling `load_func` with the
```
('will be created' right before)